src/interfaces/userInterface.ts

```diff
@@ -0,0 +1,13 @@
+export interface UserProps {
+  id: number;
+  name: string;
+  birthDate?: string;
+  position?: string;
+  salary?: number;
+  projects?: string[];
+  imageUrl?: string;
+}
+
+export interface UserCardProps {
```

Comment:
When you put this interface in the component file, don't forget to remove it from here!